test/smoke/src/sql/areas/notebook/notebookView.test.ts

```diff
@@ -9,21 +9,24 @@ import * as minimist from 'minimist';
 import { afterSuite, beforeSuite } from '../../../utils';
 
 export function setup(opts: minimist.ParsedArgs) {
-	describe('NotebookView', () => {
+	describe('NotebookTreeView', () => {
 		beforeSuite(opts);
 		afterSuite(opts);
 
 		it('Pin a notebook', async function () {
 			const app = this.app as Application;
 			await app.workbench.sqlNotebook.view.focusNotebooksView();
-			await app.workbench.sqlNotebook.view.pinNotebook();
-			await app.workbench.sqlNotebook.view.waitForPinnedNotebookView();
+			const notebookIds = await app.workbench.sqlNotebook.view.getNotebookTreeItemIds();
+			// Pinning SQL notebook to prevent the Configure Python Wizard from showing, since Python is no longer set up when the NotebookTreeView test suite starts
+			await app.workbench.sqlNotebook.view.pinNotebook(notebookIds[1]);
+			await app.workbench.sqlNotebook.view.waitForPinnedNotebookTreeView();
 		});
 
 		it('Unpin Notebook', async function () {
```

Comment:
So this can be done separately since this isn't really directly related to the fix you're doing - but right now this test isn't actually verifying anything about the unpinned Notebook which isn't great. It should at the very least be ensuring that the Notebook item being unpinned was actually removed from the list (and probably verify it shows back up in the non-pinned Notebook view).